packages/cydran/src/log/appender/ConsoleAppenderImpl.ts

```diff
@@ -1,166 +1,112 @@
-import { Appender } from "log/appender/Appender";
 import Level from "log/Level";
-import { Properties } from 'properties/Property';
-import { defaulted, isDefined, padRight } from "util/Utils";
 import SimpleMap from "interface/SimpleMap";
-import { PropertyKeys } from "CydranConstants";
-import PropertiesImpl from "properties/PropertiesImpl";
+import { DEFAULT_LOG_STRATEGY, PropertyKeys } from "CydranConstants";
+import { AbstractAppender, getNow } from "log/appender/AbstractAppender";
+import { isDefined } from "util/Utils";
+
+const doPreamble = (label: string, lvl: Level, pOrder: string[]): string => {
+	let result: string = "";
+	for(const tok of pOrder) {
+		switch(tok.toLowerCase()) {
+			case "time":
+				result += getNow() + " ";
+				break;
+			case "level":
+				result += "[" + lvl.padEnd(5, " ") + "] ";
+				break;
+			case "name":
+				result += "[ " + label + " ] ";
+				break;
+			default:
+				break;
+		}
+	};
+	return result.trim();
+}
 
 const colorPfx: string = PropertyKeys.CYDRAN_LOG_COLOR_PREFIX as const;
-const getNow = (): string => {
-	const now = new Date();
-	return `${now.getFullYear()}-${now.getMonth()}-${now.getDate()} ${now.getHours()}:${now.getMinutes()}:${now.getSeconds()}:${now.getMilliseconds()}`;
-};
+const preamOrdrKey: string = PropertyKeys.CYDRAN_LOG_PREAMBLE_ORDER as const;
+
+const PREAMBLFMT: string = "%c%s " as const;
 
 type OutColor = {orig: string, alt: string};
+type LogPrep = {color: string, preamble: string};
 
-class ConsoleAppenderImpl implements Appender {
+class ConsoleAppender extends AbstractAppender {
```

Comment:
Class name no longer agrees with the package filename.

Reply:
fixed